SmartDeviceLink/SDLPermissionManager.m

```diff
@@ -74,6 +74,15 @@ - (BOOL)isRPCAllowed:(NSString *)rpcName {
     return [item.hmiPermissions.allowed containsObject:self.currentHMILevel];
 }
 
+- (BOOL)isRPCPermitted:(SDLRPCFunctionName)rpcName {
+    if (self.permissions[rpcName] == nil || self.currentHMILevel == nil) {
+        return NO;
+    }
+
+    SDLPermissionItem *item = self.permissions[rpcName];
+    return [item.hmiPermissions.allowed containsObject:self.currentHMILevel];
+}
```

Comment:
Since this is the same implementation as `isRPCAllowed`, the deprecated one can change to just call the new one. That way if we ever have to fix a bug we don't have to fix it twice.